controllers/profiles/unfollowAnonnimousUser.js

```diff
@@ -0,0 +1,80 @@
+const moment = require('moment');
```

Comment:
Typo on file name 
should be `unfollowAnonymousUser` or should be `unfollowAnonUser` for short